### DISCUSSION TITLE
Rename master to main?

### DISCUSSION THREAD

**Discussion by bjlittle** [SciTools/nc-time-axis#65] [Q&A] [answered]
Given the migration of the community and `git` tooling away from `master` to `main`, are there any opinions on whether `nc-time-axis` should adopt this rebrand?

See [here](https://github.com/github/renaming) for further details.

Now, if any, seems like a very apt time to decide on this.

What do you think @SciTools/nc-time-axis-devs ? :thinking:

**Comment by trexfeathers** [comment, marked as answer]
Needs to happen
